Makefile

```diff
@@ -133,13 +124,13 @@ test-list: test/test-list.js
 test/test-list.js: force
 	@git ls-files -o -c test/feature | node build/build-test-list.js > $@
 
-test-interpret: test/unit/runtime/test_interpret.js
+test-interpret: test/unit/runtime/traceur-interpreter.js
 	./traceur $^
 
-test-interpret-throw: test/unit/runtime/throwsError.js
+test-interpret-throw: test/unit/runtime/resources/throwsError.js
```

Comment:
Yes, exactly.  Here I am explicitly listing the directories containing tests rather than including all `test/unit` and excluding `*/resources`. 

`test-interpret-throw` could be a node test (`generated-code-dependencies.js`)